src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Object getRandomBall() {
+        Random random = new Random();
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
Similarly, the `ColorSupplier` instance does not need to be created every time the `getRandomBall` method is called. Consider making it a private final field of the `Lottery` class and initialize it in the constructor or directly in the field declaration.